skpro/classification/sklearn_classifiers/_sklearn_proba_class.py

```diff
@@ -0,0 +1,144 @@
+"""Adapter to sklearn multiclass classification using Histograms."""
+
+__author__ = ["ShreeshaM07"]
+
+import numpy as np
+import pandas as pd
+
+from skpro.distributions import Histogram
+from skpro.regression.base import BaseProbaRegressor
+from skpro.utils.sklearn import prep_skl_df
+
+
+class SklearnProbaClassifier(BaseProbaRegressor):
```

Comment:
name should be more descriptive

Reply:
I wasn't very sure of what to name it so I just named it along the lines of `SklearnProbaReg`. I'll modify the name giving it more description.